test/test_rake_task_arguments.rb

```diff
@@ -58,6 +58,7 @@ def test_deconstruct_keys
     omit "No stable pattern matching until Ruby 3.1 (testing #{RUBY_VERSION})" if RUBY_VERSION < "3.1"
 
     ta = Rake::TaskArguments.new([:a, :b, :c], [1, 2, 3])
+    assert_equal ta.deconstruct_keys(nil), { a: 1, b: 2, c: 3 }
     assert_equal ta.deconstruct_keys([:a, :b]), { a: 1, b: 2 }
```

Comment:
```suggestion
    assert_equal({ a: 1, b: 2, c: 3 }, ta.deconstruct_keys(nil))
    assert_equal({ a: 1, b: 2 }, ta.deconstruct_keys([:a, :b]))
```

Being an RSpec devotee, I spoke with an accent when I added this test and had the `assert_equal(expected, actual)` order backwards. It'd only affect the failure message but I'd hate for someone plugging a hole to copy my bad accent...

Reply:
I'm also an rspec guy, so I get the order of these backwards pretty often myself.  Anyway, I pushed an update with them in the correct order.  Thanks. 🙂